code/game/turfs/simulated/floor/lava.dm

```diff
@@ -122,6 +122,19 @@
 		else
 			to_chat(user, "<span class='warning'>You need one rod to build a heatproof lattice.</span>")
 			return
```

Comment:
This isn't your code. but this `return` probably has to be indented one less now that we have a second thing to check in this proc